src/v/pandaproxy/schema_registry/handlers.cc

```diff
@@ -175,6 +176,36 @@ put_config_subject(server::request_t rq, server::reply_t rp) {
     co_return rp;
 }
 
+ss::future<server::reply_t>
+delete_config_subject(server::request_t rq, server::reply_t rp) {
+    parse_content_type_header(rq);
+    parse_accept_header(rq, rp);
+    auto sub = parse::request_param<subject>(*rq.req, "subject");
+
+    rq.req.reset();
+
+    // ensure we see latest writes
+    co_await rq.service().writer().read_sync();
+
+    compatibility_level lvl;
```

Comment:
Not very nice. Normally I'd put a lambda or something around the try/catch, but I don't think that works here because coro. Helper function?